engine/index.js

```diff
@@ -462,3 +462,8 @@ export default {
   buildStatus,
   buildIndex,
 };
+
+const test = {
```

Comment:
OK, let's file a follow-up to use `rewire` or `babel-plugin-rewire` once it's feasible.